doc/findinfiles.rst

```diff
@@ -1,27 +1,58 @@
-:orphan:
+####
+Find
+####
 
-#############
-Find in Files
-#############
-
-The **Find in Files** pane allows to you perform full-text, recursive content search in a user-defined scope, with advanced features to filter your results.
+The **Find** pane allows you to search the content of files in a user-defined location, with advanced features to filter your results.
 
 .. image:: images/find_in_files/find_in_files_inprogress.png
    :alt: Spyder Find in Files panel, with search results shown per-file
 
 |
 
 
+
+===================
+Using the Find pane
+===================
+
+To search for text in the Find pane, enter it in the combobox in the top left and press the search button.
```

Comment:
```suggestion
To search for text in the Find pane, enter it in the field in the top left and press the search button.
```

* I know that's technically a combobox, but people not familiar with the term would have a hard time trying to find it.

Reply:
```suggestion
To search for text in the Find pane, enter it in the field in the top left and press the search button.
```

I agree, its just that I'm pretty sure we used "combobox" in the Profiler and Pylint docs so I used it here for consistancy...

Reply:
I checked and we are not using combobox on any of those so I agree with @ccordoba12.

Reply:
Glad to know you also agree.